packages/uicore/src/styles/colors.css

```diff
@@ -163,6 +163,43 @@
   --theme-dark-canvas-dot: #585b81;
   --theme-dark-nav-panel-bg: #030d19;
 
+  /* ===== Corporate Brand Colors ===== */
+
+  /* Continuous Delivery */
+  --corporate-cd-foreground: #30841f;
+  --corporate-cd-border: #5fb34e;
+  --corporate-cd-background: #f6fff2;
+  /* Continuous Integration  */
+  --corporate-ci-foreground: #0672b6;
+  --corporate-ci-border: #2bb1f2;
+  --corporate-ci-background: #e2f5ff;
+  /* Cloud Cost Management */
+  --corporate-ccm-foreground: #24807f;
+  --corporate-ccm-border: #01c9cc;
+  --corporate-ccm-background: #ecffff;
+  /* Feature Flag Management */
+  --corporate-ff-foreground: #c05809;
+  --corporate-ff-border: #ee8625;
+  --corporate-ff-background: #fcf4e3;
+  /* Service Reliability Management */
+  --corporate-srm-foreground: #6938c0;
+  --corporate-srm-border: #ae82fc;
+  --corporate-srm-background: #f6f1ff;
+  /* Security Test Orchestration */
+  --corporate-sto-foreground: #1947ec;
+  --corporate-sto-border: #4791ff;
+  --corporate-sto-background: #ebefff;
+  /* Chaos Engineering */
+  --corporate-ce-900: #350018;
+  --corporate-ce-800: #68002d;
+  --corporate-ce-700: #9a0041;
+  --corporate-ce-600: #cd0056;
+  --corporate-ce-500: #ff006a;
+  --corporate-ce-400: #ff368a;
+  --corporate-ce-300: #ff6cab;
+  --corporate-ce-200: #ffa1cb;
+  --corporate-ce-100: #fff1f7;
```

Comment:
Do we need these? I thought we just needed a foreground/background/border per module?

Reply:
In the brand Color document, CE is the only module that has a range of colors:

<img width="563" alt="Screenshot 2022-07-13 at 16 02 05" src="https://user-images.githubusercontent.com/103499516/178766386-1b957928-5f0f-472b-a8c0-cca69ad7297b.png">

Although I could keep it in line with the other colors for now until we actually need those other colors